### DISCUSSION TITLE
DIGIT FSM Installation issue on local environment

### DISCUSSION THREAD

**Discussion by dhruvkumar-dot** [egovernments/Digit-Core#167] [Water-Sanitation]
I tried to install the DIGIT v2.8 for the FSSM module on my local system but after installation, no services are being created, and hence not able to access the application.

I have followed the steps mentioned here: https://core.digit.org/guides/installation-guide/quick-setup

After installation, 
<img width="506" alt="42958dce-796a-4d67-8d53-0718c3a8fc74" src="https://github.com/egovernments/Digit-Core/assets/58928264/3d6b0cc2-e027-416a-b67b-0189bcf6567e">

1. only 3 pods are running in Kubernetes. There are no pods running for other gov services
<img width="593" alt="4a9fee3b-2122-44b3-8d77-b67651808ae0" src="https://github.com/egovernments/Digit-Core/assets/58928264/e7da95bc-5a1f-4e46-ade6-d047abbbc9ad">

2. Tested the Digit application status in the command prompt/terminal using the command below:
curl -Is http://quickstart.local.digit/employee/login |  head -n 1
**It should show HTTP/2 200 as Ouput but getting no output**
![3e95b04c-5a20-4131-ba19-da6c0168a1a6](https://github.com/egovernments/Digit-Core/assets/58928264/ffb7cadc-c4f9-4663-9edb-a86fea0644e4)

Overall executed steps can be found here: https://docs.google.com/spreadsheets/d/1OuEbUz2KzIWzgVp4GSFIrzgkelG4jh8NSiO7dPhqG9Y/edit#gid=0

**Comment by subhashini-egov** [comment]
@talele08 have asked Nikhil to check this and respond.

**Reply by dhruvkumar-dot** [reply]
Hi @subhashini-egov @talele08 
We still have not received any solution to this issue. Please respond on this

**Reply by naresh-egov** [reply]
@dhruvkumar-dot Could you kindly provide the output generated by executing the '**sudo go run digit_setup.go**' command? This information will assist us in diagnosing the issue more effectively. Currently, none of the DIGIT pods are running on your machine, which is why you're unable to access the DIGIT service.

**Reply by naresh-egov** [reply]
@dhruvkumar-dot  For enhanced DIGIT performance, we recommend deploying it in a cloud environment(AWS, Azure)
